### PR TITLE
Remove 'Change' action for no user cases

### PR DESCRIPTION
**Pull request by yusufsheiqh**
## Context

- Remove the "Change" action link when there are no user cases
  - when an admin user invite a direct employee of their own organisation

### CODE REVIEW

File: trade_remedies_public/cases/v2_views/invite.py [hunk 1]
```diff
@@ -206,6 +206,9 @@ def get_context_data(self, **kwargs):
         context["cases"] = self.client.organisations(
             self.request.user.contact["organisation"]["id"], fields=["cases"]
         )
+        context["user_cases"] = self.client.organisations(
+            self.request.user.contact["organisation"]["id"], fields=["user_cases"]
```
Comment (chris-pettinga) [inline trade_remedies_public/cases/v2_views/invite.py:210]: Let's just use one API call and retrieve both the cases and user_cases fields, e.g.

```suggestion
            user_organisation = self.client.organisations(
            self.request.user.contact["organisation"]["id"], fields=["cases", "user_cases"]
        )
        context["cases"] = user_organisation.cases
        context["user_cases"] = user_organisation.user_cases
```

Comment (yusufsheiqh) [reply trade_remedies_public/cases/v2_views/invite.py]: Resolved in https://github.com/uktrade/trade-remedies-public/pull/255/commits/bff857b9e7238ee7a8b32ead19f01dd2c636402a.

File: trade_remedies_public/cases/v2_views/invite.py [hunk 2]
```diff
@@ -203,9 +203,11 @@ class ReviewInvitation(BaseInviteView):
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
-        context["cases"] = self.client.organisations(
-            self.request.user.contact["organisation"]["id"], fields=["cases"]
+        cases = self.client.organisations(
+            self.request.user.contact["organisation"]["id"], fields=["cases", "user_cases"]
         )
+        context["cases"] = cases
```
Comment (chris-pettinga) [inline trade_remedies_public/cases/v2_views/invite.py:209]: cases is currently an OrganisationObject, you will need to access the specific 'cases' property of that object:

```suggestion
        context["cases"] = cases.cases
```

That being said, it's probably best to rename 'cases' to something more readable, like 'organisation'

Comment (yusufsheiqh) [reply trade_remedies_public/cases/v2_views/invite.py]: Resolved in https://github.com/uktrade/trade-remedies-public/pull/255/commits/4fe509f67bcb959c2be68b341ccec99565fb5059.

**Review by chris-pettinga — APPROVED** [review]

**Pull request merged**